src/vs/workbench/browser/parts/dialogs/dialogHandler.ts

```diff
@@ -70,7 +70,7 @@ export class BrowserDialogHandler extends AbstractDialogHandler {
 
 		const buttons = this.getInputButtons(input);
 
-		const { button, checkboxChecked, values } = await this.doShow(input.type ?? 'question', input.message, buttons, input.detail, buttons.length - 1, input?.checkbox, input.inputs);
+		const { button, checkboxChecked, values } = await this.doShow(input.type ?? 'question', input.message, buttons, input.detail, buttons.length - 1, input?.checkbox, input.inputs, typeof input.custom !== 'boolean' ? input.custom : undefined);
```

Comment:
looks good, just notice that in all other cases we check against 'object' and here we check against 'boolean'

Reply:
good point, thank you!